contents/dataframesmeta_subset.md

```diff
@@ -0,0 +1,44 @@
+## Row Selection {#sec:dataframesmeta_subset}
+
+We already covered two macros that operates on _columns_, `@select` and `@transform`.
+
+Now let's cover the only macro we need to **operate on _rows_: `@subset`**
+
+It follows the same principes we've seen so far with DFM,
+_except_ that the **operation must return a boolean variable** for row selection.
```

Comment:
Maybe making this one paragraph without the newlines makes sense? I associate one-sentence paragraphs with low quality online tutorials.